charts/redpanda/ci/02-one-node-cluster-no-tls-no-sasl-values.yaml

```diff
@@ -30,3 +30,9 @@ connectors:
   enabled: true
   logging:
     level: debug
+
+# tests to ensure this large int isn't converted to scientific notation for the rpk commands
+# in post-upgrade job.
+config:
+  cluster:
+    retention_local_target_ms_default: 21600000
```

Comment:
The same apply to tiered storage

https://github.com/redpanda-data/helm-charts/blob/9744a28e9cb9dc8da141a2f2fa0a6af849ea6bb5/charts/redpanda/templates/_configmap.tpl#L290-L303
and
https://github.com/redpanda-data/helm-charts/blob/9744a28e9cb9dc8da141a2f2fa0a6af849ea6bb5/charts/redpanda/templates/_configmap.tpl#L83-L97

Reply:
Please open a new issue.